openid-4-verifiable-presentations-1_0.md

```diff
@@ -1646,7 +1646,7 @@ if (credential.protocol == "openid4vp") {
 }
 ```
 
-# Examples with Credentials in Various Formats {#alternative_credential_formats}
+# Credential Format Identifiers {#alternative_credential_formats}
```

Comment:
is this a good name for a section that explains and defines more than just client identifiers (every credential format has sections **credential identifier**, **credential metadata**, **authorization request**, etc..)

Reply:
```suggestion
# Credential Format Profiles {#alternative_credential_formats}
```

good point. we call these credential format profiles ;)